lbcapi/api.py

```diff
@@ -44,9 +51,14 @@ def call(self, method, url, params={}, stream=False, files=None):
         if url.startswith(self.server):
             url = url[len(self.server):]
 
-        # Convert parameters into list of tuples.
-        # This makes request encoding to keep the order
-        params = params.items()
+        if sys.version_info <= (3, 0):
+            # If not py3 - convert parameters into list of tuples.
+            # This makes request encoding to keep the order.
+            # In py3 "requests" lib fails with list of tuples,
+            # so the dict is necessary.
```

Comment:
I can't say that I performed all the tests as it could be, but I has been tested several GET requests ind them has been performed well. But, in case,  it would be better to test it significantly.